app/models/sign.rb

```diff
@@ -62,4 +68,8 @@ def touch_last_ping
   def active?
     last_ping && (Time.zone.now - 8.seconds) <= last_ping  # The poll is every 5 seconds (3 second delay is fine)
   end
+
+  # Alias methods generated by :visibility enum
+  def public?() listed? end
+  def private?() hidden? end
```

Comment:
Ruby also offers an [alias method](http://stackoverflow.com/questions/4763121/should-i-use-alias-or-alias-method). But I'm ok with this.